### ISSUE TITLE
Accounting Talk on Twitter: Buzz around earning announcements

### ISSUE THREAD

**Issue by BiancaMinuth** [trr266/rescorptrans#3]
Hi all,
I am very interested in this topic! If you are also curious about 'Accounting Talk on Twitter' I would be happy to collaborate on it. Feel also free to change the name of the topic or bing your own suggestions on it. 
Best
Bianca

**Comment by rkoprik** [comment]
Hey Bianca, 
I would be really interested in the topic, maybe we could partner up.
Best, 
Rebecca

**Comment by xumj2021** [comment]
I  am also interested :)

**Comment by BiancaMinuth** [comment]
Hi Mengjie and Rebecca,
seems like we are a group of 3 then. :D I am looking very much forward working with you on the Twitter project. I thought it might be nice to get to know each other a bit via a video call. Would you two be interested and if yes, when are you usually available? 
I am genereally free in the evening from 5pm (CET, UTC+2). 

Enjoy your weekend and see you on Monday. :)

Bianca

**Comment by BiancaMinuth** [comment]
**Here is my idea of the topic:**
The introduction of social media platforms such as Facebook and Twitter as disclosure channels allows firms to directly disseminate news to and communicate with investors rather than relying on the press (Miller & Skinner, 2015; Blankespoor et al., 2014 Dorminey et al., 2015). Over the last years, CEOs have become more and more comfortable using social media as a disclosure channel to disseminate firm news, like earnings announcements (Chen et al., 2019). I would be very interested to find out more about to what extent and in what way (reporting style, timing, etc.) firms use Twitter to disseminate financial information (earnings announcements). Also, I am curious to do some research on the regulation on the distribution of financial information through social media and whether there is need for clarification and stronger monitoring. 

Let me know what you are interested in!! :)

**Comment by xumj2021** [comment]
> Hi Mengjie and Rebecca,
> seems like we are a group of 3 then. :D I am looking very much forward working with you on the Twitter project. I thought it might be nice to get to know each other a bit via a video call. Would you two be interested and if yes, when are you usually available?
> I am genereally free in the evening from 5pm (CET, UTC+2).
> 
> Enjoy your weekend and see you on Monday. :)
> 
> Bianca

Hi Bianca and Rebecca,

I am generally available in the evening from 5.30 pm. Look forward to talking to you soon. :)

Best,
Mengjie

**Comment by rkoprik** [comment]
Hey Bianca and Mengjie, 

I'm excited to work with you both. I am in for the video call - I'm usually free from 5:30PM too. Let me know which day works for you both, looking forward to see you!

Regarding the idea of the topic, it sounds great! Maybe, reporting style and some sentiment analysis could possibly be combined (just a 1st thought that came to my mind). 

Have a nice weekend! :)

**Comment by BiancaMinuth** [comment]
Hi Mengjie and Rebecca,

perfect. I created a doodle: https://doodle.com/poll/x96uzwcneaxk8kny?utm_source=poll&utm_medium=link

@ Rebecca: sentiment analysis is a great idea! Sounds very exciting. 

Looking forward meeting you soon and see you on Wednesday. :)

**Comment by BiancaMinuth** [comment]
Hi Mengjie and Rebecca,

thank you for participating in the doodle survey. Based on your availabilities I created a google meet for Tuesday, May 11 5:30pm: 
https://meet.google.com/fpq-ptiz-pyh

Looking forward seeing you there. :)

Best,
Bianca

**Comment by towitter** [comment]
Hi guys! From our survey, we have the following participants that show an interest in this topic (github handle in brackets, if provided): 

mengjie (@xumj2021), David (@davidogu), Bianca (@BiancaMinuth), Rebecca (@rkoprik), Jonas (@jonwessel), Rusha (@nrusha95), Mahmoud (@mdelshadi), Gabriel (@gabriel aryanindita)

**Comment by BiancaMinuth** [comment]
Hi all,

Mengjie, Rebecca, and me had the idea to meet online before we start working on the project **Accounting Talk on Twitter**. The meeting [google meet](https://meet.google.com/fpq-ptiz-pyh) is set for next Tuesday May 11, 5:30pm CET, UTC+2. It would be great to see you there if you can make it :) https://meet.google.com/fpq-ptiz-pyh 

@ mengjie (@xumj2021), David (@davidogu), Rebecca (@rkoprik), Jonas (@jonwessel), Rusha (@nrusha95), Mahmoud (@mdelshadi), Gabriel (@gabriel aryanindita)

**Comment by nrusha95** [comment]
Hi Bianca, 
I think it is a great idea to meet once before starting to work on the project. While I will definitely attend the meeting on May 11, can you please put me in the loop for all further discussions on this project through emails, etc.?
Also, the timing is European Standard time right?
Thanks.

**Comment by BiancaMinuth** [comment]
> Hi Bianca,
> I think it is a great idea to meet once before starting to work on the project. While I will definitely attend the meeting on May 11, can you please put me in the loop for all further discussions on this project through emails, etc.?
> Also, the timing is European Standard time right?
> Thanks.

@nrusha95 Yes, CET time. Hope this works for you. And everything we have discussed so far is here on GitHub. But the communication channel a good point to discuss either here using 'issues' or during the meeting. Maybe email is easier to follow, but I am also very open for other channels like GitHub.

**Comment by nrusha95** [comment]
> > Hi Bianca,
> > I think it is a great idea to meet once before starting to work on the project. While I will definitely attend the meeting on May 11, can you please put me in the loop for all further discussions on this project through emails, etc.?
> > Also, the timing is European Standard time right?
> > Thanks.
> 
> @nrusha95 Yes, CET time. Hope this works for you. And everything we have discussed so far is here on GitHub. But the communication channel a good point to discuss either here using 'issues' or during the meeting. Maybe email is easier to follow, but I am also very open for other channels like GitHub.

Yes, the time works for me. Yes we can discuss which channels to use for further communication during the meeting on Tuesday. Look forward to talking to you al then

**Comment by jonwessel** [comment]
Hey, 
I give a tutorial on Tuesdays till 5:45 pm. I will join a bit later then but I will make it. :)

**Comment by jonwessel** [comment]
Hi Mahmoud (@mdelshadi), Hi Gabriel (@gabriel aryanindita), 

I hope you are doing well! 

Currently, we are six people, how would like to work on the topic "Accounting Talk on Twitter: Buzz around earning announcements"! If you would like to join us, please feel free to reach out to us!
